### ISSUE TITLE
WARNING:  The annotation image is not supported for generating context content

### ISSUE THREAD

**Issue by Bhargav-Ravinuthala** [ragapp/ragapp#228]
Whenever i tried to upload some image from the user need, It gives me this warning in the console and application not able to recognize the input image that is given 

**To Reproduce**
1. click on the attachment symbol 

![image](https://github.com/user-attachments/assets/6c1f8a39-645e-4cdc-a96f-252fe997f5c5)
 
2. select png/jpeg 
3. Ask a query about the image 

**Expected behavior**
Application should be able to recognize the input image

![image](https://github.com/user-attachments/assets/4f9d38f4-60b1-4335-97d3-46b129939dbc)

Tested with 
GPT 4 Turbo
Gemini 1.5 Pro

**Environments**

-  WSL
- Tested with both docker and Dev environments

**Comment by marcusschiesser** [comment]
yes, uploading images doesn't work see https://github.com/run-llama/create-llama/issues/65
